lstchain/image/muon/muon_analysis.py

```diff
@@ -125,13 +127,29 @@ def fit_muon(x, y, image, geom, tailcuts):
     image_clean: `np.ndarray`
         Image after cleaning
     """
+    
+    if tailcuts == None:
+        #we want to quantize the noise of the image. To do so, we will use the negative Q cumulative distribution.
+        negative_Q=np.sort(image[image <= 0])
+
+        hist, bins = np.histogram(negative_Q,range=(-15,0),bins=30)
+        bins=bins[:-1]
+
+        normalised_cumulative_hist = np.asarray(np.cumsum(hist)/np.sum(hist))
+
+        idx = (np.abs(normalised_cumulative_hist - 0.318)).argmin() #Find q closest to standard deviation
```

Comment:
Explain better: we want to get, from a single image, a quantity related to the width of the noise distribution, but only using the negative side of the distribution of pixel charges (because the positive side includes actual signal, i.e. the light from the muon).  So we look for the value of q below which we find 31.8% of the pixels in the image. We consider that "1 sigma" to use it as a reference to determine the image cleaning.  
"dev" is just the absolute value of that (it would correspond to the standard deviation in case the distribution was gaussian).

Reply:
transferred to #1112